Sources/TuistCore/Graph/Mappers/GraphMapper.swift

```diff
@@ -4,33 +4,23 @@ import TuistGraph
 
 /// A protocol that defines an interface to map dependency graphs.
 public protocol GraphMapping {
-    /// Given a graph, it maps it into another graph.
-    /// - Parameter graph: Graph to be mapped.
-    func map(graph: Graph) throws -> (Graph, [SideEffectDescriptor])
```

Comment:
I don't think we can get rid of this just yet - from what I gather, we first need to implement all graph mappers to work with `ValueGraph` (we can still use `map(graph:Graph)` and convert `Graph` to `ValueGraph` as the first thing in the mapper).

Before turning this switch, it's also necessary that `loadProjectWorkspace` returns `ValueGraph` instead of `Graph`.